public/style.css

```diff
@@ -47,63 +72,70 @@ header form {
   align-items: center;
 }
 
-header .right-nav {
-  flex-grow: 0;
-  white-space: nowrap;
-}
-
 header input {
   -webkit-appearance: none;
   -moz-appearance: none;
+  appearance: none;
   padding: 5px;
   background: #fff;
   border: 1px solid #cfd6e6;
   border-radius: 3px;
-  font-size: 0.95rem;
+  font-size: 1.05em;
   line-height: 1.2;
+  width: 100%;
 }
 
 header input:focus {
-  border-color: #5294e2;
+  border-color: var(--blue-300);
 }
 
 header button {
+  --icon-size: calc(1.2em + 2px);
+  --padding: 4px;
   -webkit-appearance: none;
   -moz-appearance: none;
+  appearance: none;
   margin-left: 4px;
-  padding: 5px 1ex;
-  background: #fbfbfb;
-  border: 1px solid #cfd6e6;
+  position: relative;
+  padding: var(--padding);
+  width: var(--icon-size);
+  height: var(--icon-size);
+  box-sizing: content-box;
+  background-color: #fbfbfb;
+  border: 1px solid transparent;
   border-radius: 3px;
-  font-size: 0.95rem;
+  font-size: 1.05em;
   line-height: 1.2;
+  text-indent: -9999px;
 }
 header button::-moz-focus-inner {
   border: 0;
 }
 
-header button:hover,
 header button:focus {
-  background: #fff;
-  border-color: #abb1be;
+  border-color: var(--blue-300);
 }
 
-header button:active {
-  background: #5294e2;
+header button:hover {
+  background-color: var(--blue-300);
+  border-color: transparent;
```

Comment:
Does the button visually "shrink" by that? If so, I'd rather use `var(--blue-300)` here, too.